eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -93,13 +93,17 @@ $targets = ($Configs | ConvertFrom-Json).targets
 foreach ($config in $targets) {
   if ($config.mode -eq "Preview") { $includePreview = $true } else { $includePreview = $false }
   $pkgsFiltered = $pkgs | ? { $_.IsPrerelease -eq $includePreview}
+  $suffix = ""
```

Comment:
Is there a separate change that adds this config for .NET?

Reply:
Yep! That's what I was trying to imply in the description of the PR. Should have been more explicit 👍